internal/repository/repositories.go

```diff
@@ -1,11 +1,23 @@
 package repository
 
+import (
+	"github.com/avialog/backend/internal/model"
+	"gorm.io/gorm"
+)
+
 type Repositories interface {
 }
 
 type repositories struct {
 }
 
-func NewRepositories() Repositories {
+func NewRepositories(db *gorm.DB) Repositories {
+	err := db.AutoMigrate(&model.User{}, &model.Aircraft{}, &model.Contact{},
+		&model.Flight{}, &model.Landing{}, &model.Passenger{})
+
+	if err != nil {
+		panic(err)
```

Comment:
Refactor this method to be `NewRepositories(db *gorm.DB) (Repositories, error)`. And then `return nil, err`. Try to avoid panic's in deeper logic code, `panic's` should appear on the top level of execution like in `main.go`